app/views/admin/users/_user.html.erb

```diff
@@ -13,7 +13,9 @@
     <%= content_tag(:div, content_tag(:span, user_role(user.member?)[:status]), class: "status #{user_role(user.member?)[:css]}") %>
   </td>
   <td>
-    <%= user.posts_count %>
+    <%= link_to admin_load_post_user_path(user.id),  {:remote => true, class: "btn btn-info", 'data-toggle' => "modal", 'data-target' => '#modal-user'} do %>
```

Comment:
thực ra k cần .id đâu e. nó toàn tự nhận hết ý

Reply:
vang chi